core-web/libs/dotcms-webcomponents/src/components/dot-html-to-image/dot-html-to-image.scss

```diff
@@ -1,7 +1,9 @@
 dot-html-to-image {
+    align-items: center;
+    background-color: #fafafa;
```

Comment:
use standard color ?

Reply:
mmm, this is a dotcms-webcomponents and Will suggested that color. @fmontes thoughts?

Reply:
That's the lightest gray we have in the list, go with vars.

Reply:
fixed!